samples/drivers/watchdog/boards/s32z270dc2_rtu0_r52.overlay

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2023 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		watchdog0 = &swt0;
+	};
```

Comment:
This should be part of the board files to avoid too many overlay files.

Reply:
Sorry for the late answer. I moved this into part of the board files to avoid too many overlay files as you suggest.